pubspec.yaml

```diff
@@ -3,17 +3,19 @@ description: An example app using the angular_components package.
 environment:
   sdk: '>=2.0.0-dev.3.0 <2.0.0'
 dependencies:
-  angular: 5.0.0-alpha+2
-  angular_components: 0.9.0-alpha+2
-  angular_forms: 1.0.1-alpha+2
+  angular: 5.0.0-alpha+4
+  angular_components: 0.9.0-alpha+3
+  angular_forms: 1.0.1-alpha+4
   browser: ^0.10.0
   dart_to_js_script_rewriter: ^1.0.1
-  sass_builder: ^1.1.0
+  sass_builder: 1.1.2
+dev_dependencies:
+  webdriver: ^1.2.1
+  build_runner: ^0.7.0
+  build_web_compilers: ^0.2.0
 dependency_overrides:
   # Necessary with angular: 5.0.0-alpha+1 dependency.
   analyzer: ^0.31.0-alpha.1
-dev_dependencies:
-  webdriver: ^1.2.1
 transformers:
```

Comment:
Do we still really need the transformers, especially angular and $dart2js? Are we encouraging folks to use both the usual `pub serve|build` _and_ possibly the new build system?

Reply:
My plan was to leave them until the angular transformer goes away. I'll add a similar comment  to make it clear they are not used when building with build_runner.